ABOUT_ME.md

```diff
@@ -0,0 +1,5 @@
+INTRODUCTION:-
+
+Hello everyone, my name is Prakash Kumar.I am pursuing B.Tech from Silicon Institute of Technology.
+Currently, I am in 8th semester and doing intern at Surya Digital.My hobbies are listening songs and going to gym.
+ 
```

Comment:
This does not seem like proper Markdown formatting, please make sure you use Markdown syntax to write in a markdown file.

Reply:
It's still not proper Markdown formatting. Please make sure that I don't have to point out same thing again.